### PR TITLE
Boons corrections

### PR DESCRIPTION
**Pull request by Boontato**
| AniDB | TVDB/TMDB/IMDB | Notes |
| --- | --- | --- |
| https://anidb.net/anime/3703 | https://thetvdb.com/index.php?tab=series&id=75205 | Season 2 remap |
| https://anidb.net/anime/5115 | https://thetvdb.com/index.php?tab=series&id=79839 | Season 2 to 1 remap |
| https://anidb.net/anime/8597 | https://thetvdb.com/index.php?tab=series&id=247691 | Season 0 ova remap with offset |
| https://anidb.net/anime/11086 | https://thetvdb.com/index.php?tab=series&id=305073 | Season 0 ova |
| https://anidb.net/anime/13706 | https://thetvdb.com/index.php?tab=series&id=344194 | Season 1 tvdb map |
| https://anidb.net/anime/15390 | https://thetvdb.com/index.php?tab=series&id=379918 | Season 1 tvdb map |
| https://anidb.net/anime/15744 | https://thetvdb.com/index.php?tab=series&id=389200 | Season 1 tvdb map |
| https://anidb.net/anime/15966 | https://thetvdb.com/index.php?tab=series&id=381536 | Season 1 |
| https://anidb.net/anime/15967 | https://thetvdb.com/index.php?tab=series&id=381536 | Season 2 |
| https://anidb.net/anime/17535 | https://thetvdb.com/index.php?tab=series&id=381536 | Season 3 |

implemented fix suggestions.

### CODE REVIEW

**Comment by Boontato** [comment]
Sorry I am a github noob and I didn't know how to edit in line with your suggestions and keep it within pr#204 so I just closed 204 and made a new PR

**Comment by BrutuZ** [comment]
Not a problem, it's perfectly normal to run into such turbulence when first dealing with GitHub and git in general.
For future reference, it's as simple as pushing new commit(s) to the same branch and the will update with them, it's even noted at the bottom:
> Add more commits by pushing to the **`Boons-Corrections`** branch on **Boontato/anime-lists**.

File: anime-list-master.xml [hunk 1]
```diff
@@ -45315,7 +45312,7 @@
   <anime anidbid="16510" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Kaijuu Step SDGs Daisakusen</name>
   </anime>
-  <anime anidbid="16511" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="16511" tvdbid="408325" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
```
Comment (BrutuZ) [inline anime-list-master.xml:45315]: `defaulttvdbseason` shouldn't be empty when `tvdbid` is present

File: anime-list-master.xml [hunk 2]
```diff
@@ -42161,7 +42158,7 @@
   <anime anidbid="15327" tvdbid="392491" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Azur Lane: Bisoku Zenshin!</name>
   </anime>
-  <anime anidbid="15328" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="15328" tvdbid="378034" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
```
Comment (BrutuZ) [inline anime-list-master.xml:42161]: `defaulttvdbseason` shouldn't be empty when `tvdbid` is present

File: anime-list-master.xml [hunk 3]
```diff
@@ -33048,7 +33045,7 @@
   <anime anidbid="12051" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Uchuu Senkan Yamato 2202: Ai no Senshi-tachi</name>
   </anime>
-  <anime anidbid="12053" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="12053" tvdbid="313185" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
```
Comment (BrutuZ) [inline anime-list-master.xml:33048]: `defaulttvdbseason` shouldn't be empty when `tvdbid` is present

File: anime-list-master.xml [hunk 4]
```diff
@@ -31760,7 +31757,7 @@
   <anime anidbid="11572" tvdbid="316815" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Mahou Shoujo Ikusei Keikaku</name>
   </anime>
-  <anime anidbid="11573" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="11573" tvdbid="301830" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
```
Comment (BrutuZ) [inline anime-list-master.xml:31760]: `defaulttvdbseason` shouldn't be empty when `tvdbid` is present

**Review by BrutuZ — CHANGES_REQUESTED** [review]

**Comment by Boontato** [comment]
Ok I think I got the fixes done.

**Pull request merged**